src/main/java/net/mcreator/element/GeneratableElement.java

```diff
@@ -39,7 +39,7 @@
 
 public abstract class GeneratableElement {
 
-	public static final int formatVersion = 29;
+	public static final int formatVersion = 32;

```

Comment:
32?

Reply:
I was going to explain it ;)
#2537 seems it will be merged before Wednesday, so I can continue to make changes. Then, #2813 also seems it will be merged, so it adds +2 to the converter number. This PR seems to be the next PR and so, it will require bumping the converter version a few times, but I may not be available for 2 weeks (explained on Discord a few days ago) when it will be required. I'm just taking this advance to not slow down new features in case only the icon is needed for its addition to the master.